templates/quiz/results-attempt.php

```diff
@@ -1,11 +1,12 @@
 <?php
 /**
- * Quiz Single Attempt Results
+ * Quiz Single Attempt Results.
  *
- * @since    3.16.0
- * @version  3.27.0
+ * @since 3.16.0
+ * @since [version] Hide answers if resumable attempt is incomplete.
+ * @version 3.27.0
```

Comment:
```suggestion
 * @since 3.16.0
 * @since 3.27.0 Unknown.
 * @since [version] Hide answers if resumable attempt is incomplete.
 * @version [version]
```